mne/preprocessing/tests/test_maxwell.py

```diff
@@ -0,0 +1,81 @@
+# Author: Mark Wronkiewicz <wronk@uw.edu>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import warnings
+import numpy as np
+from numpy.testing import (assert_equal, assert_allclose,
+                           assert_array_almost_equal)
+from scipy.special import sph_harm as scipy_sph_harm
+from mne.preprocessing import maxwell
+from mne.datasets import sample
+from mne.io import Raw
+from mne.datasets import testing
```

Comment:
see how we handle importing the `testing` dataset in other places and follow the ad-hoc standard, i.e..:
```
data_path = op.join(testing.data_path(download=False))
```